plugin/pkg/global/resourcereferencemanager/admission.go

```diff
@@ -862,22 +865,24 @@ func (r *ReferenceManager) ensureBindingReferences(ctx context.Context, attribut
 	)
 
 	for _, quotaRef := range quotas {
-		readAttributes := authorizer.AttributesRecord{
-			User:            attributes.GetUserInfo(),
-			Verb:            "get",
-			APIGroup:        gardencorev1beta1.SchemeGroupVersion.Group,
-			APIVersion:      gardencorev1beta1.SchemeGroupVersion.Version,
-			Resource:        "quotas",
-			Subresource:     "",
-			Namespace:       quotaRef.Namespace,
-			Name:            quotaRef.Name,
-			ResourceRequest: true,
-			Path:            "",
-		}
-		if decision, _, err := r.authorizer.Authorize(ctx, readAttributes); err != nil {
-			return fmt.Errorf("could not authorize read request for quota: %w", err)
-		} else if decision != authorizer.DecisionAllow {
-			return fmt.Errorf("%s cannot reference a quota you are not allowed to read", binding.GetObjectKind().GroupVersionKind().Kind)
+		if !isGardenadmUser(attributes.GetUserInfo()) {
```

Comment:
This allows gardenadm user to have read permission across namespaces. Is this desired?
```suggestion
		if !isGardenadmUser(attributes.GetUserInfo()) || credentialsNamespace != attributes.GetNamespace(){
```

Reply:
Good catch. Thanks, PTAL